client/src/pages/Profile/ProfileSideBar/ProfileAvailability.tsx

```diff
@@ -0,0 +1,3 @@
+export default function ProfileAvailability(): JSX.Element {
+  return <h2>Profile Availability</h2>;
```

Comment:
Use MUI `Typography` component to replace this `h2`, and ditto every other place.